routes/courseRoutes.js

```diff
@@ -54,6 +55,21 @@ router.post("/courses", async (req, res) => {
   }
 });
 
+router.get("/course/getHome", async (req, res) => {
+
+  res.send(["Hello course","Hello course 242424","Hello test"]);
+});
+
+//Route that fetched all subscribed courses of logged in user. It is using a fixed user now, but the out commented function is for the logged in user 
+router.get("/course/getSubscribedCourses", async (req, res) => {
+  const subscribedCourses = JSON.parse(JSON.stringify(await UserModel.findById('650c26466fe6094f6214a4a4', 'subscriptions -_id'))).subscriptions;
+  //const subscribedCourses = JSON.parse(JSON.stringify(await UserModel.findById(req.user.id, 'subscriptions -_id'))).subscriptions;
+  const list = await CourseModel.find({'_id': { $in: subscribedCourses }});
+  console.log(list)
```

Comment:
Any reason for this console log? If there is I'd suggest making the logging more clear, not just printing a list.